repo/maintenancestats/stats_clean_up_log.go

```diff
@@ -0,0 +1,41 @@
+package maintenancestats
+
+import (
+	"fmt"
+
+	"github.com/kopia/kopia/internal/contentlog"
+)
+
+const cleanupLogsStatsKind = "cleanupLogsStats"
+
+// CleanupLogsStats are the stats for cleanning up logs.
+type CleanupLogsStats struct {
+	ToDeleteBlobCount int   `json:"toDeleteBlobCount"`
+	ToDeleteBlobSize  int64 `json:"toDeleteBlobSize"`
+	DeletedBlobCount  int   `json:"deletedBlobCount"`
```

Comment:
At present, `Deletedxxx` always equals to `ToDeletexxx` if dryRun is FALSE. But it hasn't to be in the case of partially successful in deletion.
To support future enhancement and consistent with other stats, I added both of them use them appropriately in different scenarios.